api-flask/app/zonal_stats.py

```diff
@@ -16,6 +21,43 @@
 DEFAULT_STATS = ['min', 'max', 'mean', 'median']
 
 
+def get_wfs_response(wfs_params, zones):
+    """Execute wfs request to external service."""
```

Comment:
Let's detail a bit more what this does. Not everyone will be familiar with a "WFS Response"

Reply:
Done. Included in 3661a96c70180094c380a44d99547340ee32811a